src/WebJobs.Extensions.DurableTask/LinuxAppServiceLogger.cs

```diff
@@ -58,9 +59,11 @@ internal class LinuxAppServiceLogger
             this.writeToConsole = writeToConsole;
             this.roleInstance = JToken.FromObject("App-" + containerName);
             this.tenant = JToken.FromObject(tenant);
+            this.stamp = stampName;
+
+            var finalCharIndex = stampName.Length - 1;
+            this.primaryStamp = char.IsLetter(stampName[finalCharIndex]) ? stampName.Remove(finalCharIndex) : stampName;
```

Comment:
Hey @davidmrdavid, so having though about this a bit more, this logic will be problematic for ASEs. Take a look at the results of this Kusto query to see what I mean:
```kusto
DurableFunctionsEvents
| where TIMESTAMP > ago(3d)
| summarize by EventPrimaryStampName, EventStampName
| order by EventPrimaryStampName desc, EventStampName asc
| take 500
```
I think it's fine to ship as-is for now since I don't expect a lot of Linux ASE usage (at least not right away), but let's open a new issue to track fixing this for a subsequent release.

Reply:
Will do!